core/src/main/java/io/fabric8/maven/core/service/kubernetes/JibBuildService.java

```diff
@@ -65,7 +64,7 @@ public void build(ImageConfiguration imageConfiguration) {
            jibBuildConfiguration = JibBuildServiceUtil.getJibBuildConfiguration(config, buildImageConfiguration, fullName, log);
            JibBuildServiceUtil.buildImage(jibBuildConfiguration, log);
        } catch (Exception ex) {
-           throw new UnsupportedOperationException();
```

Comment:
Re-throwing exceptions without the cause should be **illegal**! 😩

Reply:
aah, really sorry! my bad